scripts/db_init.py

```diff
@@ -29,7 +29,7 @@
         "Name varchar(255) NOT NULL," \
         "Date TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP()," \
         "Status enum('active', 'done', 'deleted', 'archive') NOT NULL DEFAULT 'active'," \
-        "Calendar_date varchar(15)," \
+        "Calendar_date DATETIME DEFAULT '0000-00-00 00:00:00'," \

```

Comment:
Sorry, It is not truth

![image](https://user-images.githubusercontent.com/24716957/43994178-3d880746-9da1-11e8-88e0-783aa615a2b3.png)

Reply:
We have the HUGE issue )))

- the File:  sql/entities.py

- ```
OperationalError: (_mysql_exceptions.OperationalError) (1048, "Column 'Calendar_date' cannot be null") [SQL: u'INSERT INTO tasks (`UserID`, `Name`, `Date`, `Status`, `Calendar_date`) VALUES (%s, %s, %s, %s, %s)'] [parameters: (3, u'Caledar date', None, 'active', None)] (Background on this error at: http://sqlalche.me/e/e3q8)
```

Reply:
Which version database do we have now ? ))) 
![image](https://user-images.githubusercontent.com/24716957/43999193-4b6caf64-9e10-11e8-9f98-73e79c635b9a.png)
![image](https://user-images.githubusercontent.com/24716957/43999184-17fdac14-9e10-11e8-8909-deefc449c275.png)

Reply:
Please, check  it - http://docs.sqlalchemy.org/en/latest/orm/extensions/declarative/table_config.html#using-a-hybrid-approach-with-table
```
Base.metadata.reflect(some_engine)
class User(Base):
    __table__ = metadata.tables['user']
```